docs/_static/js/version-alert.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+// Source:
+// https://github.com/anymail/django-anymail/blob/4c443f5515d1d5269a95cb54cf75057c56a3b150/docs/_static/version-alert.js
+
+function warnOnLatestVersion() {
+
+  // The warning text and link is really specific to RTD hosting,
+  // so we can just check their global to determine version:
+  if (!window.READTHEDOCS_DATA || window.READTHEDOCS_DATA.version !== "latest") {
+    return;  // not latest, or not on RTD
+  }
+
+  var warning = document.createElement('div');
```

Comment:
Maybe it would be simpler to add the warning to the html template and use a css selector or inline style with `display: none`?
Then this code could just replace the link and set an inline style to `display: block`.

Reply:
Okay, I'll look into the sphinx documentation whether we can override the theme we are using.

Reply:
This is fine if it's too confusing.